content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1118,3 +1118,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 
 * The [deprecation policy](/docs/reference/using-api/deprecation-policy/) for Kubernetes explains
   the project's approach to removing features and components.
+* Starting in Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta
+  feature, you will also need to enable any associated API resources.  This is congruent to alpha
+  features today.  For example, to enable a particular resource like
```

Comment:
Omit the “This is congruent to alpha features today” because “today” anchors the docs in time. We prefer to write timeless documentation (see https://developers.google.com/style/timeless-documentation for Google's take on this).